### PR TITLE
Add eqMac 2.0.7.

### PR DESCRIPTION
**Pull request by sepehr**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

### CODE REVIEW

**Comment by vitorgalvao** [comment]
We can change its token to `eqmac` if we merge https://github.com/caskroom/homebrew-cask/pull/37416.

File: Casks/romankisil-eqmac.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+cask 'romankisil-eqmac' do
+  version '2.0.7'
+  sha256 'caa97a681371f60173aabd3949e381f8b53234fb9642340d49795a495df0fa15'
+
+  # github.com/romankisil/eqMac2 was verified as official when first introduced to the cask
+  url "https://github.com/romankisil/eqMac#{version.major}/releases/download/v#{version}/eqMac#{version.major}.dmg"
```
Comment (sepehr) [inline Casks/romankisil-eqmac.rb]: What has happened to the url here? @vitorgalvao

File: Casks/romankisil-eqmac.rb [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
   # github.com/romankisil/eqMac2 was verified as official when first introduced to the cask
   url "https://github.com/romankisil/eqMac#{version.major}/releases/download/v#{version}/eqMac#{version.major}.dmg"
-  appcast 'https://github.com/romankisil/eqMac#{version.major}/releases.atom',
+  appcast "https://github.com/romankisil/eqMac#{version.major}/releases.atom",
```
Comment (sepehr) [inline Casks/romankisil-eqmac.rb]: This is also buggy.

**Comment by commitay** [comment]
> `github.com/romankisil/eqMac` does not exists.

No, but I don't think we can use `#{version.major}` in a commented line. 
____

Please rebase and rename the cask to `eqmac`, thanks!

**Comment by sepehr** [comment]
> No, but I don't think we can use #{version.major} in a commented line.

Seems like that the developer has chose this as part of the application name: "eqMac2". eqMac is the name for its initial release as the [homepage comparison table](https://www.bitgapp.com/eqmac/) shows.

So what should we do? eqmac or eqmac2?

@commitay

**Comment by commitay** [comment]
To follow the [token_reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md#simplified-names-of-apps) the cask should be `eqmac`

**Comment by sepehr** [comment]
Rebased and squashed.

**Review by commitay — APPROVED** [review]

**Pull request merged**